system_test/helpers/aest_nodes.erl

```diff
@@ -355,3 +345,21 @@ decode_json(Data) ->
     catch
         error:badarg -> {error, {bad_json, Data}}
     end.
+
+scan_logs_for_errors() ->
+    Logs = aest_nodes_mgr:get_log_paths(),
+    maps:fold(fun(NodeName, LogPath, Result) ->
+        LogFile = binary_to_list(filename:join(LogPath, "epoch.log")),
+        case filelib:is_file(LogFile) of
+            false -> Result;
+            true ->
+                Command = "grep '\\[error\\]' '" ++ LogFile ++ "'",
```

Comment:
This is a start though I am concerned that check relies a lot on this regexp/string. Not sure if there are any other meaningful strings.

I wonder if there are alternatives:
* Review that all nodes are configured to log crash.log, and check it is empty file (so not need to grep);
* As Operator may want, enable by config logging all log entries not meant to exist (error or higher?) in separate log file then check that log file is empty?